hunting-buddy.lic

```diff
@@ -207,32 +207,38 @@ class HuntingBuddy
       @exit = [escort_info['area'], 'exit']
     else
       @exit = nil
-      find_empty_room(rooms, waiting_room,
```

Comment:
Enable "Hide whitespace changes" when reviewing this diff otherwise it looks like more changed than what actually did.

The lambda didn't need to be indented so far to the right and that's the whitespace changes.